packages/cli/src/commands/join.ts

```diff
@@ -92,6 +94,24 @@ export async function handleJoin(argv: JoinArgv, packageVersion: string) {
     )
   );
 
+  if (!argv.decorate) {
```

Comment:
Taking into account [this](https://github.com/Redocly/redocly-cli/issues/1042#issuecomment-1503120311), how do you think about implementing this logic in the `bundle` command itself, so we can pass the `decorate` option there and avoid code duplication?

Reply:
We can move this code in future. When we have concrete vision for that task